### ISSUE TITLE
Simple quote en fin des champs texte (nomEntreprise, commune, ...)

### ISSUE THREAD

**Issue by adriens** [adriens/emploi-nc-api#69]
![image](https://user-images.githubusercontent.com/5235127/88591597-4f324880-d0a8-11ea-950b-9c422bcea052.png)

**Comment by zYmMiJ** [comment]
Je l'ai mis a jour j'étais pourtant sur de l'avoir fait en meme temps que le sdk.

![image](https://user-images.githubusercontent.com/43323367/88628310-79602680-d0f9-11ea-95f4-90b096a302ac.png)

**Comment by adriens** [comment]
Du coup c'est normal les guillemets  ❔

**Comment by zYmMiJ** [comment]
yep c'est ce ticket la : https://github.com/adriens/emploi-nc-api/issues/54

**Comment by adriens** [comment]
✅
